__snapshots__/index.test.js.snap

```diff
@@ -1318,22 +1318,70 @@ exports[`should pass on packages/eslint-config-base/test/samples/es2015-esm/cont
 Array [
   Object {
     "column": 1,
-    "line": 5,
+    "line": 4,
     "rule": "strict",
     "severity": 2,
   },
   Object {
-    "column": 13,
+    "column": 1,
+    "line": 8,
+    "rule": "import/order",
+    "severity": 2,
+  },
+  Object {
+    "column": 8,
+    "line": 8,
+    "rule": "no-unused-vars",
```

Comment:
This shouldn't happen. Can you add a something with the `index` variable to suppress this error?